README.md

```diff
@@ -12,7 +12,7 @@
 Run this in a shell and leave it running:
 
 ```
-$ docker-compose up
+$ docker-compose up --build --detach
```

Comment:
I don't think we want to detach by default. The Rails server has good output.

Reply:
you can get the output with 

    docker-compose logs --follow

would that do?